frontend/e2e/tests/versioning-tests.ts

```diff
@@ -20,13 +20,13 @@ import Project from '../../common/project';
 export default async () => {
     await flagsmith.init({fetch,environmentID:Project.flagsmith,api:Project.flagsmithClientAPI})
     const hasFeature = flagsmith.hasFeature("feature_versioning")
+    log('Login')
```

Comment:
```suggestion
    console.log('Login')
``` 

Not sure if this is correct or not but seems consistent with the statement below.